test/kv.go

```diff
@@ -215,7 +215,7 @@ func create(kv kvdb.Kvdb, t *testing.T) {
 		"Expected action KVCreate, actual %v", kvp.Action)
 
 	_, err = kv.Create(key, []byte("bar"), 0)
-	assert.Error(t, err, "Create on existing key should have errored.")
+	assert.Error(t, kvdb.ErrExist, "Create on existing key should have errored.")
```

Comment:
Should not this be

if (assert.Error(t, err, "...") {
    assert.Equal(t, kvdb.ErrExist, err)
}